src/selectize.js

```diff
@@ -231,7 +231,8 @@ $.extend(Selectize.prototype, {
 					return false;
 				}
 				// blur on click outside
-				if (!self.$control.has(e.target).length && e.target !== self.$control[0]) {
+				// do not blur if the dropdown is clicked
+				if (!self.$dropdown.has(e.target).length && e.target !== self.$control[0]) {
```

Comment:
`e.target` is always contained within `self.$dropdown` here.  `$dropdown` and `$control` are sibling elements in the DOM.   `$dropdown` is the correct element to use for this logic.